modules/mobile/app/models/mobile/v0/adapters/lighthouse_individual_claims.rb

```diff
@@ -112,25 +112,42 @@ def events_timeline(attributes)
 
           # sort to put events with uploaded == false on top and then by date
           events.compact.sort_by do |event|
-            upload_priority = if event[:uploaded] || !event.key?(:uploaded)
-                                0 # Lower priority for uploaded == true or key not present
+            upload_priority = if event[:uploaded] || event.uploaded.nil?
```

Comment:
Will event always respond to `uploaded` here? Just want to check if there's the chance it will error out

Reply:
all events in that array should now be instances of the model so it'll always have `uploaded`.